src/components/DataDictionary.tsx

```diff
@@ -123,14 +124,16 @@ export const columns: ColumnDescriptor[] = [
     name: "type",
     label: "Type",
     rowValueDescriptor: "type",
-    formatter: (x: any) => humanize(x),
+    Formatter: function RenderType({ x }: FormatterProps) {
+      return <>{humanize(x)}</>
```

Comment:
check type of `x`

if `humanize()` returns a string, you can just `return humanize(x)`